src/particles/beam/BeamParticleContainer.cpp

```diff
@@ -202,7 +199,7 @@ BeamParticleContainer::InitData (const amrex::Geometry& geom)
               amrex::ParallelDescriptor::Communicator());
 #endif
 
-    if (Hipace::HeadRank()) {
+    if (Hipace::HeadRank() && m_injection_type != "fixed_ppc") {
         m_init_sorter.sortParticlesByBox(*this, geom);
     }
```

Comment:
Why do we still sort particles by box when we removed the loop over boxes with the per slice comms?

Reply:
The function actually sorts per slice (1 box == 1 slice there)